docs/draft/sticker-pack.md

```diff
@@ -0,0 +1,77 @@
+---
+title: IPFS gateway for Sticker Pack
+version: 0.1.0
+status: Draft
+authors: Gheorghe Pinzaru <gheorghe@status.im>
+redirect_from:
+  - /sticker-pack.html
+---
+
+## Table of Contents
+
+ 1. [Abstract](#abstract)
+ 2. [Specification](#specification)
+ 3. [Copyright](#copyright)
+
+## Abstract
+
+In this specification, we describe how Status app uses IPFS gateway to store stickers.
+We will explore image format, how they are uploaded and how an end user can see them
+inside the Status app.
+
+## Definition
+
+| Term             | Description                                                                            |
+|------------------|----------------------------------------------------------------------------------------|
+| **Stickers**     | A set of images which can be used to express emotions                                  |
+| **Sticker Pack** | ERC721 token which includes the set of stickers                                        |
+| **IPFS**        | P2P network used to store and share data, in this case, the images for the stickerpack |
+
+## Specification
+
+### Image format
+Accepted image file types are **PNG, JPG/JPEG and GIF**, with a maximum allowed size of 300kb.
+The minimum sticker image resoulution is 512x512, and background of clipart should be transparent.
+
+### Distribution
+
+Sticker pack are implemented as ERC721 token and contains a set of stickers. These stickers
+are stored inside the sticker pack as a set of hyperlinks pointing to IPFS storage. These
+hyperlinks are publically available and can be accessed by any user inside the status chat.
+Stickers can be sent in chat only by accounts that own the sticker pack.
+
+### IPFS gateway
+At the moment of writing, Status uses the [Infura](https://infura.io/) gateway.
```

Comment:
Rephrase as "current main Status app" is using Infura. And something like "However clients could choose to run IPFS node..."~